fsp_email.py

```diff
@@ -13,13 +13,13 @@
 import subprocess
 import fsp_reservation
 
-_RE_crew1 = re.compile("^\s*Pilot: (.+?)\s*$")
-_RE_crew2 = re.compile("^\s*Instructor: (.+?)\s*$")
-_RE_crew3 = re.compile("^\s*For: (.+?)\s*$")
-_RE_aircraft = re.compile("^\s*Aircraft: .*?(\S+)\s*$")
-_RE_start = re.compile("^\s*Resource Start Time: (\d+)/(\d+)/(\d+) (\d+):(\d+) ([AP])M\s*$")
-_RE_end = re.compile("^\s*Resource End Time: (\d+)/(\d+)/(\d+) (\d+):(\d+) ([AP])M\s*$")
-_RE_resv_id = re.compile("^\s*Reservation Id: (\d+)\s*$")
+_RE_crew1 = re.compile("^\s*\*For: (.+?)\*\s*$")
```

Comment:
\* (backslash star) is not valid in a Python string. By sheer luck it happens to work anyway because, quoting [the docs](https://docs.python.org/2.0/ref/strings.html), "Unlike Standard C, all unrecognized escape sequences are left in the string unchanged, i.e., *the backslash is left in the string*." meaning the string value contains the backslash, therefore the regex engine sees the backslash, and does the right thing. Solutions:

1. (Recommended) use Python raw strings like r"this" or like r'this' when the contents are meant to be regular expressions. That way, the backslashes get included literally in the strings by *design* instead of by *accident* and the regex engine sees them.

2. Quote the backslashes like this: "foo \\* bar" so that Python string processing eats one backslash and the regex engine sees the other.

Applies to the following lines as well (some of them were already wrong, btw).